neuraxle/pipeline.py

```diff
@@ -72,15 +72,16 @@ def transform(self, data_inputs: Any):
         """
         self.setup()
 
-        current_ids = self.hasher.hash(
-            current_ids=None,
-            hyperparameters=self.hyperparams,
-            data_inputs=data_inputs
-        )
-        data_container = DataContainer(current_ids=current_ids, data_inputs=data_inputs)
-        data_container = self._transform_core(data_container)
-
-        self.teardown()
+        try:
```

Comment:
Why do a try catch here? It should be a quite good reason, otherwise please don't.

Reply:
To be precise, it's fine for a pipeline to fail. The catch must be placed inside the AutoMLWrapper or in the RandomSearch class for example.

Reply:
not supposed to be in this pr

Reply:
PLEASE OPEN ISSUE BEFORE RESOLVING if you resolve.